include/mbedtls/ssl.h

```diff
@@ -2533,13 +2533,19 @@ void mbedtls_ssl_conf_cert_req_ca_list( mbedtls_ssl_config *conf,
 #if defined(MBEDTLS_SSL_MAX_FRAGMENT_LENGTH)
 /**
  * \brief          Set the maximum fragment length to emit and/or negotiate
- *                 (Default: the smaller of MBEDTLS_SSL_IN_CONTENT_LEN and
+ *                 (Typical: the smaller of MBEDTLS_SSL_IN_CONTENT_LEN and
  *                 MBEDTLS_SSL_OUT_CONTENT_LEN, usually 2^14 bytes)
  *                 (Server: set maximum fragment length to emit,
- *                 usually negotiated by the client during handshake
+ *                 usually negotiated by the client during handshake)
  *                 (Client: set maximum fragment length to emit *and*
  *                 negotiate with the server during handshake)
  *
+ * \note           By default the \c mfl_code field of the \c mbedtls_ssl_config
+ *                 structure is equal to `0 == MBEDTLS_SSL_MAX_FRAG_LEN_NONE`.
+ *                 Therefore, the maximum fragment length extension *will not*
+ *                 be used, unless the maximum fragment length has been set to a
+ *                 different value via this function.
```

Comment:
I think we need to clarify that this is about clients. Unless I'm mistaken, servers will accept the extension regardless of that setting (as long as support is enabled at compile-time).